utility/SSD_writeback_cache.py

```diff
@@ -0,0 +1,64 @@
+import json
+import logging
+import os
+import random
+import subprocess
+import sys
+import time
+
+log = logging.getLogger(__name__)
+
+logging.basicConfig(
+    handlers=[logging.StreamHandler(sys.stdout)],
+    level=logging.INFO,
+    format="%(asctime)s - %(levelname)s - %(filename)s:%(lineno)d - %(message)s",
+)
+
+
+def ssd_write_cache():
+    """
+
+    This method to validate constant flushing and
+    evicting of data for an extended period of time.
+    This scenario in particular
+    recommend writing a script that would run through a few thousand
+    iterations of SIGKILLed "rbd bench" "fio" on the same image:
+
+    """
+    size = 1 * 1024 * 3
+    pool_name = "ssd"
+    image_name = "image3"
```

Comment:
Would it make sense to retrieve these values from CLI? 

The current implementation creates the required resources but does not cleanup after the workflow completion. Can we implement a teardown sequence?

Reply:
https://github.com/red-hat-storage/cephci/blob/master/pipeline/scripts/ci/cos_cli.py please refer this for custom values which should be picked from CLI.

Reply:
Yes, implemented via cli with custom values as per the reference provided